### PR TITLE
easytable:0.1.0

### PR DESCRIPTION
**Pull request by monaqa**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: a simple package for writing tables in Typst.
This aims for clear, straightforward markup and some flexibility.

```typst
#easytable({
  th[Header 1 ][Header 2][Header 3  ]
  td[How      ][I       ][want      ]
  td[a        ][drink,  ][alcoholic ]
  td[of       ][course, ][after     ]
  td[the      ][heavy   ][lectures  ]
  td[involving][quantum ][mechanics.]
})
```

image

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
    - Is "easytable" not obvious?
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/easytable/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Typst Easytable Package
+
+A Typst library for writing simple tables.
+
+## Usage
+
+**WARNING: This library is not in `preview` namespace.**
```
Comment (laurmaedje) [inline packages/preview/easytable/0.1.0/README.md:7]: This warning should be removed before submission to the preview namespace.

**Review by laurmaedje — COMMENTED** [review]
The name is fine. :)

**Comment by emilyyyylime** [comment]
and wouldn't those be better described by `tr`?

**Comment by monaqa** [comment]
Thank you for your comments, I removed the warning and added some explanation for elements (`td`, `th`, and so on).

**Comment by laurmaedje** [comment]
I think what @emilyyyylime wanted to say is that `td` (table data) in the code should be `tr` (table row) because it describes a row rather than a cell.

**Comment by monaqa** [comment]
Sorry, I misunderstood.

I wondered whether to use `th` & `td` pair or `th` & `tr` pair too, and I chose `th` & `td` in terms of the contrast between Header Cells and Data Cells.
If the combination of `th` and `tr` is comfortable for many people, I think it could be a pair of `th` and `tr`. What do you think?

**Comment by laurmaedje** [comment]
I think by using the HTML terms there is an expectation that it works like HTML, so I would expect `tr` for a row.

**Comment by monaqa** [comment]
Okay, renamed: `td` -> `tr`.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**